app/containers/Avatar.js

```diff
@@ -40,6 +41,9 @@ const Avatar = React.memo(({
 		uri = formatUrl(`/avatar/${ room }`, baseUrl, uriSize, avatarAuthURLFragment);
 	}
 
+	function openView() {
+		Navigation.navigate('AttachmentView', { attachment: { image_url: uri, image_type: 'png' } });
```

Comment:
We're sending `width=100&height=100` here.
When we want to see an image on a modal, we want it as bigger as possible.

Reply:
So the `AttachmentView` opens the image using the full width of the device but due to avatar being square does not cover the whole height.
I'll still change the size to make sure the image rendered is not pixelated.